modules/rules_poetry/0.2.0/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(
+    name = "rules_poetry",
+    version = "0.2.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "platforms", version = "0.0.6")
+
+internal_deps = use_extension("@rules_poetry//python:extensions.bzl", "internal_deps")
+internal_deps.install()
```

Comment:
just a note: this tag is unnecessary. You can also delete the tag class from your extension definition

Reply:
(resolving since this shouldn't be a blocker, and requires changes in the original project)